[Notifications] Add latest set of alerting notifications (#11175)
cloudflare/cloudflare-docs: content/notifications/notification-available.md

</details>

<details>
<summary>Magic Network Monitoring: DDoS Attack</summary>
<div>

**Who is it for?**

BYOIP customers and Spectrum customers with Network Analytics who want to receive a notification when Cloudflare has mitigated an attack.

**Other options / filters**

None.

**Included with**

Purchase of Magic Transit and/or BYOIP.

**What should you do if you receive one?**

{{<render file="_ddos.md">}}

</div>
</details>

<details>
<summary>Magic Network Monitoring: Volumetric Attack</summary>
<div>

**Who is it for?**

> Magic Transit On Demand customers who are using Flow Based Monitoring to detect attacks when Magic Transit is disabled.

Reviewer: ```suggestion
[Magic Transit on-demand](/magic-transit/on-demand/) customers who are using Flow-based Monitoring to detect attacks when Magic Transit is disabled.
```

**Other options / filters**

None.

**Included with**

> Purchase of Magic Transit.

Reviewer: ```suggestion
Purchase of [Magic Transit](/magic-transit/).
```

**What should you do if you receive one?**

> If you do not have auto advertisement enabled, you need to advertise your IP prefixes to enable Magic Transit. For more information, see [Dynamic advertisement](/byoip/concepts/dynamic-advertisement/).

Reviewer: ```suggestion
If you do not have auto advertisement enabled, you need to advertise your IP prefixes to enable Magic Transit. For more information, refer to [Dynamic advertisement](/byoip/concepts/dynamic-advertisement/).
```

</div>
</details>

## Traffic Monitoring

<details>
<summary>Advanced Error Rate Alert</summary>
<div>

**Who is it for?**

Enterprise customers who want to receive a notification when Cloudflare detects edge and/or origin errors.

**Other options / filters**

Multiple filters available:
- Customers can search and add domains from their list of domains.
- Customers can filter alerts by
  - Edge status code
  - Origin status code
  - IP Address
- Customers can also choose the trigger that fires the notification. Available triggers are:
  - Low sensitivity
  - Medium sensitivity
  - High sensitivity
  - Very High sensitivity

**Included with**

Enterprise plans.

**What should you do if you receive one?**

1. Use the link in the Notification you received to see which error codes Cloudflare is seeing.

> 2. Depending on the statuses you're alerting on, check https://developers.cloudflare.com/support/troubleshooting/cloudflare-errors/troubleshooting-cloudflare-5xx-errors/

Reviewer: ```suggestion
2. Depending on the statuses you are alerting on, refer to [Troubleshooting Cloudflare 5xx errors](/support/troubleshooting/cloudflare-errors/troubleshooting-cloudflare-5xx-errors/).
```

3. {{<render file="_errors.md">}}

</div>
</details>

<details>
<summary>Origin Error Rate Alert</summary>
<div>

**Who is it for?**

Enterprise customers who want to receive a notification when Cloudflare is unable to access their origin server.

**Other options / filters**

Multiple filters available:

- Customers can search and add domains from their list of domains.
- Customers can also choose the trigger that fires the notification. Available triggers are:
  - Low sensitivity
  - Medium sensitivity
  - High sensitivity
  - Very High sensitivity

**Included with**

Enterprise plans.

**What should you do if you receive one?**

1. Use the link in the Notification you received to see which error codes Cloudflare is seeing from your origin.
2. {{<render file="_errors.md">}}

</div>
</details>

<details>
<summary>Passive Origin Monitoring</summary>
<div>

**Who is it for?**

> Any customer who wants to receive a notification when Cloudflare is unable to access their origin, customers will only receive this notification when their origin is returning a `521` error.

Reviewer: ```suggestion
Any customer who wants to receive a notification when Cloudflare is unable to access their origin. Customers will only receive this notification when their origin is returning a `521` error.
```

**Other options / filters**

None.

**Included with**

All Cloudflare plans.

**What should you do if you receive one?**

{{<render file="_errors.md">}}

</div>
</details>

<details>
<summary>Traffic Anomalies Alert</summary>
<div>

**Who is it for?**

Enterprise customers who want to receive a notification  when one of your zones is experiencing an unexpected spike or drop in traffic.

**Other options / filters**

Multiple filters available:
- Customers can search and add domains from their list of domains.
- Customers can include or exclude traffic mitigated by the Web Application Firewall.
- Customers can choose whether to be notified of either spikes or drops in traffic.

**Included with**

Enterprise plans.

**What should you do if you receive one?**

> 1. Use the link in the Notification you received to view if the spike or drop is significant enough to require further actions

Reviewer: ```suggestion
1. Use the link in the Notification you received to view if the spike or drop is significant enough to require further actions.
```

2. {{<render file="_errors.md">}}

</div>
</details>

## Trust and Safety Blocks

<details>
<summary>Block Review Rejection</summary>
<div>

**Who is it for?**

Any customer who wants to be notified when Cloudflare Trust & Safety rejects a request for block removal.

**Other options / filters**

None.

**Included with**

> All Cloudflare plans.

Reviewer: ```suggestion
All plans.
```

**What should you do if you receive one?**

> Take care of any abuse on your website. Then, in Cloudflare go to the dashboard and request review.

Reviewer: ```suggestion
Take care of any abuse on your website. Then, go to the [Cloudflare dashboard](https://dash.cloudflare.com/) and request a review.
```

</div>
</details>

<details>
<summary>New Blocks</summary>
<div>

**Who is it for?**

> Any customer who wants to be notified when Cloudflare Trust & Safety places a block in one of your websites.

Reviewer: ```suggestion
Any customer who wants to be notified when Cloudflare Trust & Safety places a block on one of your websites.
```

> Any customer who wants to be notified when Cloudflare Trust & Safety places a block in one of your websites.

Reviewer: ```suggestion
Any customer who wants to be notified when Cloudflare Trust & Safety places a block in one of their websites.
```

**Other options / filters**

None.

**Included with**

All Cloudflare plans.

**What should you do if you receive one?**

Take care of any abuse on your website. Then, in Cloudflare go to the dashboard and request review.

</div>
</details>

<details>
<summary>Removed Blocks</summary>
<div>

**Who is it for?**

Any customer who wants to be notified when Cloudflare Trust & Safety removes a block from your website.

**Other options / filters**

None.

**Included with**

All Cloudflare plans.

**What should you do if you receive one?**

> This is informational followup.

Reviewer: ```suggestion
This is informational follow up.
```

</div>
</details>

## WAF

<details>
<summary>Advanced Security Events Alert</summary>
<div>

**Who is it for?**

> Enterprise customers who want to receive alerts about spikes in specific services that generate log entries in security events.

Reviewer: ```suggestion
Enterprise customers who want to receive alerts about spikes in specific services that generate log entries in [Security Events](/waf/security-events/paid-plans/).
```

For more information, refer to [WAF alerts](/waf/reference/alerts/).

**Other options / filters**

- Customers can search for and add domains from their list of enterprise zones.
- Customers can choose which services the alert should monitor (Managed Firewall, Rate Limiting, etc.).

> - Customers can filter events by a targetted action

Reviewer: ```suggestion
- Customers can filter events by a targeted action.
```

**Included with**

Enterprise plans.

**What should you do if you receive one?**

Review the information in [Security Events](/waf/security-events/paid-plans/) to identify any possible attack or misconfiguration.

</div>
</details>

<details>
<summary>Security Events Alert</summary>
<div>

**Who is it for?**

> Business and Enterprise customers who want to receive alerts about spikes across all services that generate log entries in security events.

Reviewer: ```suggestion
Business and Enterprise customers who want to receive alerts about spikes across all services that generate log entries in [Security Events](/waf/security-events/paid-plans/).
```

For more information, refer to [WAF alerts](/waf/reference/alerts/).

**Other options / filters**

- Customers can search for and add domains from their list of business or enterprise zones. The notification will be sent for the domains chosen.

> - Customers can filter events by a targetted action

Reviewer: ```suggestion
- Customers can filter events by a targeted action.
```
